Document that HomeKit emits an event (#21189)
home-assistant/home-assistant.github.io: source/_integrations/homekit.markdown

    packet: XXXXXXXX
```

## Events

> This integration emits events as `homekit_state_change` on the event bus. This can be used for example in automations to known whether an entity's state got changed from HomeKit specifically. 

Reviewer: Maybe an example event would be useful here?

Reviewer 2: Unresolving this comment.

@parautenbach You marked this comment as resolved without a response or other actions taken. Care to share why you marked it resolved?

Author: It must've been by accident, sorry. I have every intention to add an example. I thought of it initially, but I'm not sure how complete examples should be. Dealing directly with the event bus almost allows one to do anything, so I was thinking of just having the trigger part of an automation as an example (vs at least a trigger and action, which is the minimum). Anyway, I'll try to have a look tonight (I forgot about this a little after seeing the notification) and then you guys can tell me whether it's good or not! :-)

    packet: XXXXXXXX
```

## Events

> This integration emits events as `homekit_state_change` on the event bus. This can be used for example in automations to known whether an entity's state got changed from HomeKit specifically. 

Reviewer: ```suggestion
The HomeKit integration emits `homekit_state_change` events. These events can be used in automations to know when an entity's state was changed from HomeKit.
```
